### ISSUE TITLE
Error when using async/await

### ISSUE THREAD

**Issue by rodelrebucas** [firebase/firebase-functions#426]
I can't deploy my function there is an error that says:

> functions: Error from emulator. mod.hasOwnProperty is not a function

and

>ReferenceError: regeneratorRuntime is not defined

when I don't include 

>  "@babel/plugin-transform-runtime"

I have this on my .babelrc 

```
{
  "plugins": [
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-transform-runtime"
  ],
  "presets": ["@babel/preset-env", "@babel/preset-flow"]
}
```
I'm using express with functions, works fine when using express, but when deploying it shows the error above.

It fails when I use async/await.

**Comment by kevinajian** [comment]
Hey Rodel,

Can you fill out the bug template with the required info including what versions you're using? async / await is only supported in Node 8, but it's hard to tell without more info.

Thanks.

**Comment by rodelrebucas** [comment]
Thanks for responding @kevinajian, I have resolve the issue now. 

For the 1st error: 

> functions: Error from emulator. mod.hasOwnProperty is not a function

The issue is I have multiple exports in a file containing the my main function.  I have exported the main express app for development and the main function app for production.

For the 2nd error:

> ReferenceError: regeneratorRuntime is not defined

I have targeted node version 8  for the `@babel/preset-env` in my `.babelrc`

I'm closing this issue now. Thanks.
